readme.md

```diff
@@ -103,6 +103,41 @@ $publisher->getEditions();
 ```
 $publisher->getEdition(12345);
 ```
+#### Create Edition
+```
+$publisher->createEdition([
+    "name" => "My Edition",
+    "onDeviceName" => "My Edition",
+    "shortName" => "MY1",
+    "targetState" => [
+        "web" => "LIVE",
+    ],
+    "documentUrl" => "https://s3-eu-west-1.amazonaws.com/my.account/test/example.pdf",
+    "publicationNodeId" => "4325325",
+    "pageBillingType" => "PLATFORM"
+]);
+```
+#### Update Edition
+```
+$publisher->updateEdition(6894180, [
+    "name" => "My Edition",
+    "onDeviceName" => "My Edition",
+    "shortName" => "My Edition",
+    "drmEnabled" => "true",
+    "iosSaleEnabled" => "true",
```

Comment:
In the docs for the API, the name of this parameter is slightly different (iosSaleOption). https://github.com/yudugit/rest-api-documentation/pull/19/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8.